### DISCUSSION TITLE
Accessing ash to enable raytracing

### DISCUSSION THREAD

**Discussion by notdanilo** [gfx-rs/wgpu#3416] [Q&A]
Is it possible to access the underlying ash backend to access its raytracing functionalities?

**Comment by JMS55** [comment]
I don't know about raytracing specifically, but there are a series of `as_hal()/as_hal_mut()` methods to get the underlying ash object. Not all objects currently have these methods, I'm working on adding some more in https://github.com/gfx-rs/wgpu/pull/3376.
